src/pages/BookDetails/BookDetails.js

```diff
@@ -1,7 +1,31 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import bookIMG from "../../assets/images/book_thumbnail.jpeg"
+import Typography from "@mui/material/Typography";
+
 import classes from "./BookDetails.module.scss";
 
 const BookDetails = () => {
-    return <h1>Book details page!</h1>
+    return (
+        <Box sx={{ flexGrow: 1 }}>
+            <Grid className={classes.gridContainer} container spacing={{ xs: 2, md: 3 }} columns={{ xs: 4, sm: 4, md: 12 }}>
+                <Grid item xs={4} sm={4} md={4} key={1} sx={{textAlign:'center'}}>
+                    <img src={bookIMG} width='50%' alt="Book Thumbnail"></img>
```

Comment:
when there is nothing between the html tags use the shorthand version:
```suggestion
                    <img src={bookIMG} width='50%' alt="Book Thumbnail" />
```